wallet/api/client_send_transaction.go

```diff
@@ -135,7 +135,7 @@ func (h *ClientSendTransaction) Handle(ctx context.Context, rawParams jsonrpc.Pa
 	h.interactor.Log(ctx, traceID, SuccessLog, "The transaction passes the anti-spam rules.")
 
 	// Sign the payload.
-	rawInputData := wcommands.ToInputData(request, stats.LastBlockHeight)
+	rawInputData := wcommands.ToInputData(request, stats.LastBlockHeight, uint64(ttl))
 	inputData, err := commands.MarshalInputData(rawInputData)
 	if err != nil {
 		h.interactor.NotifyError(ctx, traceID, InternalError, fmt.Errorf("could not marshal input data: %w", err))
```

Comment:
Why is the code from `CheckTransaction` not added here ?

```go
if rawInputData.GoodForBlocks == 0 || rawInputData.GoodForBlocks > stats.MaxTTL {
    rawInputData.GoodForBlocks = stats.MaxTTL
}
```

Reply:
knew I was likely to forget something, will add it